app/helpers/application_helper.rb

```diff
@@ -165,4 +165,10 @@ def get_next_id(object)
 
     query.result_ids[idx + 1] || query.result_ids[idx - 1]
   end
+
+  # make View components easier to call
+  def component(name, *args, **kwargs, &block)
```

Comment:
Depending on how you and others feel about all-arguments forwarding, 
the following should work here. Your choice:
```suggestion
  def component(name, ...)
```